.travis.yml

```diff
@@ -18,10 +18,10 @@ jobs:
     include:
 
         - stage: Test
-          name: Symfony 4.0
+          name: Symfony 4.1
```

Comment:
This is the problem with using the full stack, individual components are versioned (and `symfony/translation` won't work with `4.0`). Would prefer a way to use each component at that version where possible.

Reply:
Nope: https://travis-ci.com/libero/content-api-bundle/jobs/175361549

Reply:
Wondered if https://github.com/symfony/flex/pull/412 would fix, but didn't (https://travis-ci.com/libero/content-api-bundle/jobs/175370448). Opened https://github.com/symfony/flex/issues/473.